render.js

```diff
@@ -0,0 +1,16 @@
+var render = (function() {
+
+  var renderMovieInfo = function(response) {
+
+  }
+
+  var renderMovieGIF = function(response) {
+
+  }
+
+  return {
+    MovieInfo: MovieInfo,
+    MovieGIF: MovieGIF
```

Comment:
you need to also update the return object since you have changed the function names.